.gitlab-ci.yml

```diff
@@ -152,13 +152,33 @@ build-windows:
   only:                            *releaseable_branches
   variables:
     CARGO_TARGET:                  x86_64-pc-windows-msvc
-    CARGO_HOME:                      "${CI_PROJECT_DIR}/.cargo"
+    CARGO_HOME:                    "C:/ci-cache/parity-ethereum/cargo/%CI_JOB_NAME%"
+    RUSTC_WRAPPER:                 sccache
+    SCCACHE_DIR:                   "C:/ci-cache/parity-ethereum/sccache"
   script:
+    - sccache -s
     - sh scripts/gitlab/build-windows.sh
+    - sccache -s
   tags:
     - rust-windows
   <<:                              *collect_artifacts
 
+# test-windows:
+#   stage:                           test
+#   only:
+#     - master
+#   variables:
+#     CARGO_TARGET:                    x86_64-pc-windows-msvc
+#       CARGO_HOME:                    C:/ci-cache/parity-ethereum/cargo/%CI_JOB_NAME%
+#       RUSTC_WRAPPER:                 sccache
+#       SCCACHE_DIR:                   C:/ci-cache/parity-ethereum/sccache
+#   script:
+#     - sccache -s
+#     - sh scripts/gitlab/test-linux.sh
+#     - sccache -s
+#   tags:
+#     - rust-windows
```

Comment:
this will be enabled after `test-windows` will be fixed.

Reply:
just remove it?

Reply:
It'll be necessary to check if tests are OK by those who will take care about them right after this PR will be merged.
I'm going to raise an issue asking for fixing windows tests and mentioning this job there, to make it comfortable.
Are you okay with that or should I create a separate PR?

Reply:
We don't want to run tests on Windows, do we?

Reply:
It was agreed that we want them nightly. Please speak straight if you have some objections.

Reply:
removed/